apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/ErrorCode.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Grpc.Core;
+
+namespace Google.Cloud.Spanner
+{
+    /// <summary>
+    /// </summary>
+    public enum ErrorCode
```

Comment:
The problem I have with the vkit namespace is that as soon as the user adds it in a using statement, a ton of conflicting crud will show up in intellisense that he should NOT use.  Stuff like session creation, executing a query without streaming, etc.  Probably half of the vkit API should not be used directly.
So until we improve the surface area of the vkit namespace, I really don't want to leak it into user code at all.  Let's set up a meeting where we can hash this out...